### PR TITLE
Update viselements file

### PR DESCRIPTION
**Pull request by dinhlongviolin1**

### CODE REVIEW

**Review by FredLL-Avaiga — APPROVED** [review]
I guess you need to modify src/assets/find_element_file.py also for:
1. look in the new path if it is not found in the old one
2. find the taipy viselements.json path too

**Comment by dinhlongviolin1** [comment]
> I guess you need to modify src/assets/find_element_file.py also for:
> 
> 1. look in the new path if it is not found in the old one
> 2. find the taipy viselements.json path too

Discussed with Fabien. I will not do this right now since it is more complicated than it looks.

**Review by FredLL-Avaiga — APPROVED** [review]

**Review by FabienLelaquais — APPROVED** [review]
Looks good.
Let's not forget to update the local viselements.json when we remove all those TODOs...
👌

File: src/assets/viselements.json [hunk 1]
```diff
@@ -1091,6 +1091,252 @@
...
+                {
+                  "name": "class_name",
+                  "type": "dynamic(str)",
+                  "doc": "The list of CSS class names that will be associated with the generated HTML Element.<br/>These class names will be added to the default <code>taipy_gui_core-&lt;element_type&gt;</code>."
+                },
+                {
+                    "name": "scenario",
+                    "default_property": true,
+                    "type": "Scenario",
+                    "doc": "TODO - The scenario to display/edit."
```
Comment (FredLL-Avaiga) [inline src/assets/viselements.json:1172]: I thought we got rid of the Todos :-(

**Review by FredLL-Avaiga — APPROVED** [review]
👍

**Pull request merged**